backend/config.default.yaml

```diff
@@ -99,7 +99,6 @@ migration:
 
 feature-flag:
   disabled-authz: true
-  public_sharing_enabled: false
```

Comment:
Removed, now not in use.  it was used https://github.com/VertaAI/modeldb/pull/1773/files#diff-40269480ca4116bb3990b71cde32415e1ae1ca3c37c7de6c1d5fbf8e38738f97L168-R158 only.